assets/css/src/amp-validation-error-taxonomy.scss

```diff
@@ -59,13 +69,15 @@ details[open] .details-attributes__summary {
 .details-attributes__summary::after,
 .single-error-detail-summary::after,
 .toggle-stylesheet-details::after {
-	order: 99;
-	width: 12px;
-	height: 12px;
 	background-image: url("../images/down-triangle.svg");
-	background-size: cover;
 	background-position: center;
+	background-size: cover;
 	content: "";
+	height: 6px;
+	position: absolute;
+	right: 0;
+	top: 7px;
+	width: 12px;
```

Comment:
This is now causing the "triangle" for stylesheet summary details to be misplaced:


https://user-images.githubusercontent.com/16200219/115323210-322e3000-a177-11eb-85f2-e42ee014f999.mp4

Reply:
Maybe this arrow can be removed, since the summary marker should be enough of an indicator in this case.

Reply:
The original implementation removes the summary marker and uses the chevron instead. In 38d1104 this has been reverted:

<img width="347" alt="Screenshot 2021-04-20 at 11 45 14" src="https://user-images.githubusercontent.com/478735/115376525-0f2f6a80-a1cf-11eb-9bd7-4f97e1aeede4.png">

What's more, for some reason the `<strong>` element not always got the `source` class applied. This, in turn, caused a slight misalignment between icons in the "Sources" column. With 218eeef this is fixed:

| Before | After |
| --- | --- |
| <img width="138" alt="Screenshot 2021-04-20 at 11 54 59" src="https://user-images.githubusercontent.com/478735/115376992-7b11d300-a1cf-11eb-9996-2a5bfa9d25d6.png"> | <img width="143" alt="Screenshot 2021-04-20 at 11 55 08" src="https://user-images.githubusercontent.com/478735/115377011-806f1d80-a1cf-11eb-8588-a1d29e69c6e3.png"> |